lib/Net/Async/Trello.pm

```diff
@@ -109,8 +109,10 @@ Returns a L<Future>.
 sub board {
 	my ($self, %args) = @_;
     my $id = delete $args{id};
+    my $uri = URI->new($self->base_uri . 'board/' . $id);
+    $uri->query_param(%args);
```

Comment:
Thanks - I need to change this slightly since `->query_param` applies a single key:

```
$uri->query_param($key => @values)
```

so this would only work for a single entry in %args.

```suggestion
    $uri->query_param($_ => $args{$_}) for keys %args;
```